js/apps/admin-ui/src/clients/advanced/OpenIdVerifiableCredentials.tsx

```diff
@@ -0,0 +1,78 @@
+import { FormGroup, Switch, Text, Button } from "@patternfly/react-core";
+import { useTranslation } from "react-i18next";
+import { HelpItem } from "@keycloak/keycloak-ui-shared";
+import { FormAccess } from "../../components/form/FormAccess";
+import { Controller, useFormContext } from "react-hook-form";
+import { convertAttributeNameToForm } from "../../util";
+import { FormFields } from "../ClientDetails";
+import { AdvancedProps } from "../AdvancedTab";
+
+export const OpenIdVerifiableCredentials = ({
+  save,
+  reset,
+  client,
+}: AdvancedProps) => {
+  const { t } = useTranslation();
+  const { control } = useFormContext();
+
+  return (
+    <>
+      <Text className="pf-v5-u-pb-lg">
+        {t("openIdVerifiableCredentialsHelp")}
+      </Text>
+      <FormAccess role="manage-clients" isHorizontal>
+        <FormGroup
+          label={t("oid4vciEnabled")}
+          fieldId="kc-oid4vci-enabled"
+          labelIcon={
+            <HelpItem
+              helpText={t("oid4vciEnabledHelp")}
+              fieldLabelId="oid4vciEnabled"
+            />
+          }
+        >
+          <Controller
+            name={convertAttributeNameToForm<FormFields>(
+              "attributes.oid4vci.enabled",
+            )}
+            defaultValue={client.attributes?.["oid4vci.enabled"] || "false"}
+            control={control}
+            render={({ field }) => (
+              <Switch
+                id="kc-oid4vci-enabled"
+                label={t("on")}
```

Comment:
I have doubts about the consistency of the ID field. Could you confirm you aligned it with what you notice in other components?
```suggestion
              <Switch
                id="oid4vciEnabled-switch"
                label={t("on")}
```